compconfig/compconfig/functions.py

```diff
@@ -30,9 +34,10 @@ def get_inputs(meta_params_dict):
     meta_params = MetaParams()
     meta_params.adjust(meta_params_dict)
     params = Specifications()
+    params.specification(serializable=True)
     spec = params.specification(
         meta_data=True,
-        year=meta_params.start_year
+        year=meta_params.year
```

Comment:
```suggestion
        serializable=True,
        year=meta_params.year
```

Reply:
Ok- tried this and still failing.  Do I need something similar in other functions?  Others create an instance of `Specifications`, but do not then create an object  named `spec`...

Reply:
Suggested one more change. Hopefully, that resolves the issue.